src/main/java/lotto/utility/LottoWinningStaticsCalculator.java

```diff
@@ -0,0 +1,57 @@
+package lotto.utility;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoWinningStaticsCalculator {
+    private static final int[] PRIZE = {0, 0, 0, 5000, 50000, 1500000, 2000000000};
```

Comment:
언더스코어(`_`)를 통해 단위가 높은 숫자의 가독성을 높여보면 좋을 것 같습니다.
```suggestion
    private static final int[] PRIZE = {0, 0, 0, 5_000, 50_000, 1_500_000, 2_000_000_000};
```